gemma-core/src/main/java/ubic/gemma/model/expression/experiment/FactorValueValueObject.java

```diff
@@ -42,6 +42,12 @@ public class FactorValueValueObject extends IdentifiableValueObject<FactorValue>
     private String factorValue;
     private String value;
     private String valueUri;
+
+    /*
+     * TODO: support objects and predicates for extension of the characteristics
+     */
+
+
```

Comment:
When including those in the VO, let's make sure we annotate the fields with `@GemmaWebOnly`.